timeline-chart/src/time-graph-state-controller.ts

```diff
@@ -24,8 +24,8 @@ export class TimeGraphStateController {
 
     constructor(protected canvas: HTMLCanvasElement, protected unitController: TimeGraphUnitController) {
```

Comment:
TIL: is this really a way to assign a new field directly from the constructor's signature?? just putting protected makes this.canvas available?

Reply:
I found that weird too. Assumed that it is canonical Javascript.